hs_core/views/__init__.py

```diff
@@ -458,10 +458,10 @@ class FilterForm(forms.Form):
 
 @processor_for('my-resources')
 def my_resources(request, page):
-    # import sys
-    # sys.path.append("/home/docker/pycharm-debug")
-    # import pydevd
-    # pydevd.settrace('172.17.42.1', port=21000, suspend=False)
+    import sys
```

Comment:
Hi folks. We need this merge today. Please be a little more forgiving on
some of these things. - Dan

On Fri, Dec 11, 2015 at 7:19 AM Hong Yi <notifications@github.com> wrote:

> In hs_core/views/__init__.py
> <https://github.com/hydroshare/hydroshare/pull/772#discussion_r47358782>:
>
> > @@ -458,10 +458,10 @@ class FilterForm(forms.Form):
> >
> >  @processor_for('my-resources')
> >  def my_resources(request, page):
> > -    # import sys
> > -    # sys.path.append("/home/docker/pycharm-debug")
> > -    # import pydevd
> > -    # pydevd.settrace('172.17.42.1', port=21000, suspend=False)
> > +    import sys
>
> this "import sys" is used by pycharm-debug and is an artifact for keeping
> debug traces in, hence is not needed. Just revert back to where it was in
> the latest develop, in fact, there is no change needed in view/*init*.py
> from this branch.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hydroshare/hydroshare/pull/772/files#r47358782>.
>